backend/spec/features/admin/users_spec.rb

```diff
@@ -152,6 +152,31 @@
       expect(user_a.reload.bill_address.address1).to eq "1313 Mockingbird Ln"
     end
 
+    context 'invalid entry' do
+      around do |example|
+        ::AlwaysInvalidUser = Class.new(Spree.user_class) do
+          validate :always_invalid_email
+          def always_invalid_email
+            errors.add(:email, "is invalid")
+          end
+        end
+        orig_class = Spree.user_class
+        Spree.user_class = "AlwaysInvalidUser"
+
+        example.run
+
+        Spree.user_class = orig_class.name
+        Object.send(:remove_const, "AlwaysInvalidUser")
```

Comment:
If the example fails (e.g. raises an exception) this cleanup won't happen. It might be better to `before` and `after` blocks, or else use `begin...ensure`?